### PR TITLE
558 chained actions

### PR DESCRIPTION
**Pull request by avdata99**
Continue from #574
Related to #558 
Replace for #601
 
Finish migration for top chained actions and auths

### CODE REVIEW

File: ckanext/unhcr/auth.py [hunk 1]
```diff
@@ -370,8 +369,6 @@ def access_request_update(context, data_dict):
     elif request.object_type == 'user':
         return external_user_update_state(context, {'id': request.object_id})
 
-    raise toolkit.Invalid("Unknown Object Type")
```
Comment (chris48s) [inline ckanext/unhcr/auth.py:373]: Everything else in this PR is fine, but I think we need to do _something_ here.

We sometimes invoke auth functions from places other than the correspondingly named action.
For example
- https://github.com/okfn/ckanext-unhcr/blob/53dbd720cd65c2cf25adc157b2eaa95b05227d31/ckanext/unhcr/auth.py#L308-L313
- https://github.com/okfn/ckanext-unhcr/blob/53dbd720cd65c2cf25adc157b2eaa95b05227d31/ckanext/unhcr/auth.py#L191-L196
- https://github.com/okfn/ckanext-unhcr/blob/53dbd720cd65c2cf25adc157b2eaa95b05227d31/ckanext/unhcr/auth.py#L235-L240
etc

If we completely remove the fallback case here and invoke this auth function somewhere else with an invalid `object_type` then this function will (implicitly) return `None`, which is not right. I don't think there is any issue with an auth function throwing `Invalid` tbh, but if you think it is wrong for the auth function to throw `Invalid` we should at least `return {'success': False}` in the fallback case.

Comment (avdata99) [reply ckanext/unhcr/auth.py:373]: Ok, fixed [here](https://github.com/okfn/ckanext-unhcr/pull/613/commits/fd799f0334e30ca36598e29f33389355821f923d)

**Comment by chris48s** [comment]
Does this mean we've now fully addressed #558 or is there still more to do?
If we're done with #558 lets close it..

**Comment by avdata99** [comment]
We are done @chris48s. I'm closing #558

**Pull request merged**